client/web/src/enterprise/campaigns/detail/ClosedNotice.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import classNames from 'classnames'
+import InformationOutlineIcon from 'mdi-react/InformationOutlineIcon'
+import { CampaignFields } from '../../../graphql-operations'
+
+interface ClosedNoticeProps {
+    closedAt: CampaignFields['closedAt']
+    className?: string
+}
+
+export const ClosedNotice: React.FunctionComponent<ClosedNoticeProps> = ({ closedAt, className }) => {
+    if (!closedAt) {
+        return <></>
+    }
+    return (
+        <div className={classNames('alert alert-info', className)}>
+            <InformationOutlineIcon className="icon-inline" /> Information on this page may be out of date because
+            changesets which only exist in closed campaigns are not synced with the code host.
```

Comment:
Absolute pedantry, sorry:

```suggestion
            changesets that only exist in closed campaigns are not synced with the code host.
```

Reply:
Ha! I knew it. I thought the same thing when reading the `which` in Figma, but then started doubting my non-native-English skills.